docs/use-cases/one-to-many.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "How to configure multiple similar machines"
+linkTitle: "Configure many machines"
+weight: 40
+type: "docs"
+tags: ["data management", "data", "services"]
+images: ["/platform/fleet.svg"]
+description: "How to configure multiple similar or identical machines with fragments."
+---
+
+Viam has a built-in tool called _{{< glossary_tooltip term_id="fragment" text="fragments" >}}_ for copying parts of a configuration file to multiple machines, as well as for modifying just some sections of a config if your machines are similar but not identical.
+
+When you update a fragment, it updates the configurations of all machines that use that fragment.
+
+{{< alert title="In this page" color="tip" >}}
+
+1. [Create a configuration fragment](#create-a-fragment)
+1. [Apply a fragment to multiple machines](#add-a-fragment-to-multiple-machines)
+1. [Modify an otherwise identical configuration](#modify-a-fragment)
+
+{{< /alert >}}
+
+For information on provisioning many machines, see [Provisioning](/fleet/provision/).
+
+## Create a fragment
+
+{{< table >}}
+{{% tablestep link="/configure/" %}}
+{{<imgproc src="/use-cases/one-to-many/config.png" resize="700x" class="fill alignleft" style="max-width: 250px" declaredimensions=true alt="Configuration builder UI">}}
+**1. Configure one machine**
+
+Start by configuring one of your machines.
+
+In the [Viam app](https://app.viam.com), use the **CONFIGURE** tab to build a configuration for all components and services you want to use on all your machines.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+{{<imgproc src="/use-cases/one-to-many/raw-json.png" resize="700x" class="fill alignleft" style="max-width: 250px" declaredimensions=true alt="JSON subtab of the CONFIGURE tab">}}
+**2. Copy the raw JSON**
+
+In your machine's **CONFIGURE** tab, switch to **JSON** and copy the raw JSON.
+
+{{% /tablestep %}}
+{{% tablestep link="/fleet/fragments/#create-a-fragment" %}}
+{{<imgproc src="/use-cases/one-to-many/new-fragment.png" resize="700x" class="fill alignleft" style="max-width: 250px" declaredimensions=true alt="app.viam.com/fragment interface">}}
+**3. Create a fragment**
+
+Go to [app.viam.com/fragments](https://app.viam.com/fragments).
+
+Add a fragment, and paste the copied JSON configuration into it.
```

Comment:
Please add a note about selecting visibility of the fragment, see DOCS-2250

Reply:
See next line. Or you'd like more than "Set your privacy settings"?

Reply:
Okay I migrated the 3 options from the reference page